### PR TITLE
Task conv-qual

### PR DESCRIPTION
**Pull request by Tako-San**

### CODE REVIEW

**Comment by Tako-San** [comment]
Try to use this one anywhere: https://en.cppreference.com/w/cpp/ranges/slide_view

File: l1/conv-qual/tokenizing.cc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+auto tokenize(std::string_view sv) {
+  auto view =
+      views::transform(sv, [](auto c) { return std::isspace(c) ? ' ' : c; }) |
+      views::split(' ') | views::filter([](auto w) { return !w.empty(); }) |
+      views::transform([](auto w) { return str2tok(w); }) | views::join;
+
+  std::vector<Token> tokens{};
+  rng::copy(view, std::back_inserter(tokens));
+  return tokens;
```
Comment (derzhavin3016) [inline l1/conv-qual/tokenizing.cc:64]: nit: `std::vector` has a ctor which accepts range as an argument

Comment (derzhavin3016) [reply l1/conv-qual/tokenizing.cc:64]: Moreover, consider using this: https://en.cppreference.com/w/cpp/ranges/to

Comment (Tako-San) [reply l1/conv-qual/tokenizing.cc]: Addressed

File: l1/conv-qual/tokenizing.cc [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+using namespace std::literals;
+
+namespace l1 {
+
+std::vector<Token> tokenize(std::string_view sv) {
+  auto tokens =
+      vws::transform(sv, [](auto c) { return std::isspace(c) ? ' ' : c; }) |
+      vws::split(' ') | vws::filter([](auto w) { return !w.empty(); }) |
+      vws::transform([](auto w) { return str2tok(w); }) | vws::join |
+      rng::to<std::vector>();
```
Comment (Tako-San) [inline l1/conv-qual/tokenizing.cc:15]: You may turn it to view

**Pull request merged**